config/scenic.lua

```diff
@@ -0,0 +1,27 @@
+ScenicRoute = {
```

Comment:
This is showing up as a new file. I think you might need to move it to the routes directory.